taskcluster/tc-build-utils.sh

```diff
@@ -276,11 +276,30 @@ do_deepspeech_netframework_build()
     /p:Platform=x64 \
     /p:TargetFramework="uap10.0" \
     /p:OutputPath=bin/nuget/x64/uap10.0
+	
+  MSYS2_ARG_CONV_EXCL='/' "${MSBUILD}" \
+    DeepSpeechClient/DeepSpeechClient.csproj \
+    /p:Configuration=Release \
+    /p:Platform="Any CPU" \
+    /p:TargetFramework="netcoreapp3.1" \
+    /p:OutputPath="bin/nuget/Any CPU/netcoreapp3.1"
+
+  MSYS2_ARG_CONV_EXCL='/' "${MSBUILD}" \
+    DeepSpeechClient/DeepSpeechClient.csproj \
+    /p:Configuration=Release \
+    /p:Platform="Any CPU" \
+    /p:TargetFramework="netstandard2.1" \
+    /p:OutputPath="bin/nuget/Any CPU/netstandard2.1"
```

Comment:
nit: space in paths are always painful, can we avoid?

Reply:
done